torch/autograd/profiler.py

```diff
@@ -231,6 +244,19 @@ def total_average(self):
         total_stat.key = 'Total'
         return total_stat
 
+    def without_nested_events(self):
```

Comment:
I don't see how this API is used, since this is a public API let's keep it to a minimum and add things only when necessary